src/main/java/ch/wisv/areafiftylan/users/controller/UserRestController.java

```diff
@@ -148,6 +149,15 @@ public User getUserById(@PathVariable Long userId) {
         return createResponseEntity(HttpStatus.OK, "User disabled");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
```

Comment:
I would change this checking method, both the `ADMIN` and `COMMITTEE` role should have access.